pure.zsh

```diff
@@ -114,13 +132,15 @@ prompt_pure_preprompt_render() {
 		# only redraw if preprompt has changed
 		[[ "${prompt_pure_last_preprompt}" != "${preprompt}" ]] || return
 
-		# calculate length of preprompt for redraw purposes
-		local preprompt_length=$(prompt_pure_string_length $preprompt)
-		local lines=$(( ($preprompt_length - 1) / $COLUMNS + 1 ))
+		# calculate preprompt length and store it in prompt_pure_preprompt_length
+		prompt_pure_string_length_to_var "${preprompt}" "prompt_pure_preprompt_length"
```

Comment:
Hmm, how else would you get the value out of the `prompt_pure_string_length_to_var` function? The other alternative would be to move the contents of `prompt_pure_string_length_to_var` inline here and remove that function.